llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -267,7 +267,7 @@ int main(int argc, char *argv[]) {
       Builder.push(File.second, TreeEntry::Regular, File.first());
     CASID SummaryID = ExitOnErr(Builder.create(*CAS));
     SummaryIDs.emplace_back(SummaryID);
-    MSG("summary tree: ");
+    MSG("summary tree: \n");
```

Comment:
Why?

Reply:
I should have explained, if you notice the test I wrote, I grep for the cas ID and then pass it along to llvm-cas. 

It is easier to isolate the cas ID itself if it is just on the next line. I couldn't think of anything and I thought this was a simple change that doesn't hurt anything

Reply:
It makes the output less compact, which will be a bit annoying when trying to scroll through a few results when running stats manually. Not the end of the world if we had to do it, but I think it's unnecessary.

In the test, you should use `-silent`, which turns off all the `MSG` stuff. Then you can do something like:
```
RUN: llvm-cas-object-format -silent >file.casid
RUN: llvm-cas @file.casid
```
and you don't need to do a `grep`.